[BUG] Compilation error on flutter beta or master: Navigator.pop(context) has type void

The app seems to be working fine on `stable` branch of flutter, but attempt to do `flutter run`  on `master`(8c0b43bda4e5ff29d399f06ce1c0e1027268d6b5) or `beta`(2294d75bfa8d067ba90230c0fc2268f3636d7584) gives the following compilation error:

```
[+4132 ms] 
                    Compiler message:
[        ]
../../.pub-cache/hosted/pub.dartlang.org/flutter_hud-0.2.1/lib/src/popup_hud.dar
           t:57:31: Error: This expression has type 'void' and can't be used.
[        ]   bool dismiss() => Navigator.pop(context);
[        ]                               ^
```

Actually, I'm also struggling to get the `develop` branch to work on the current `beta` (0.15). 

```
== Building for Android ==

> /usr/local/bin/flutter build appbundle --debug
Running Gradle task 'bundleDebug'...                            


Compiler message:

../programs/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_hud-0.2.1/lib/src/popup_hud.dart:57:31: Error: This expression has type 'void' and can't be used.

  bool dismiss() => Navigator.pop(context);

                              ^

Target kernel_snapshot failed: Exception: Errors during snapshot creation: null

build failed.



FAILURE: Build failed with an exception.



* Where:

Script '/Users/builder/programs/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 833



* What went wrong:

Execution failed for task ':app:compileFlutterBuildDebug'.

> Process 'command '/Users/builder/programs/flutter/bin/flutter'' finished with non-zero exit value 1
```

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -21,7 +21,10 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_bloc: ^3.2.0
-  flutter_hud: ^0.2.1
+  flutter_hud:
+    git:
+      url: https://github.com/eyro-labs/flutter_hud.git
+      ref: 0d5ed29fc2aa97dff2863ab824dba48264048c6a
   flutter_launcher_icons: ^0.7.3
   flutter_markdown: ^0.3.4
   flutter_phoenix: ^0.1.0

